lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,133 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
+
+import { screen } from '@testing-library/react';
+import { rtlRenderElement, rtlRender } from '../../../__testutils__/rtlUtils';
 
 describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+  const renderEl = rtlRenderElement(NxPageHeader, {});
+  const quickRender = rtlRender(NxPageHeader, {});
+
+ it('renders a top-level banner element', function() {
+    const el = renderEl();
+    const header = screen.getByRole('banner');
+    expect(header).toBe(el);
   });
 
-  it('passes its links, homeLink, and children props to the AbstractNxPageHeader', function() {
-    const props = {
-          links: [{ name: 'foo', href: '#bar' }],
-          homeLink: '#home',
-          children: <button>Click Here For a Free iPhone</button>
-        },
-        component = getShallowComponent(props);
+  it('renders the default logo within the header', function() {
+    const view = quickRender(),
+      logo = view.getByRole('img');
+    expect(logo).toBeInTheDocument();
+  });
+
+  it('renders a custom logo instead of the default one when given logo prop', function() {
+    const view = quickRender({ logo: { path: 'foo', alt: 'bar' } }),
+      logo = view.getByRole('img');
+
+    expect(logo).toHaveAttribute('src', 'foo');
+    expect(logo).toHaveAttribute('alt', 'bar');
```

Comment:
The global sidebar PR made me think of this but it applies here too.  Is it better to test the alt attribute specifically or just that this text becomes the logo's accessible name?

Reply:
> The global sidebar PR made me think of this but it applies here too. Is it better to test the alt attribute specifically or just that this text becomes the logo's accessible name?

Checking by logo's accessible name is slightly better than checking alt. I made the change here: [2fa9aa3](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/2fa9aa37e99f43f7a3ed1ba836ec8199c45fb5bb)